node/wipe_appwrite_bucket/src/index.js

```diff
@@ -0,0 +1,70 @@
+const sdk = require("node-appwrite");
+
+/*
+  'req' variable has:
+    'headers' - object with request headers
+    'payload' - request body data as a string
+    'variables' - object with function variables
+
+  'res' variable has:
+    'send(text, status)' - function to return text response. Status code defaults to 200
+    'json(obj, status)' - function to return JSON response. Status code defaults to 200
+
+  If an error is thrown, a response with code 500 will be returned.
+*/
+
+module.exports = async function (req, res) {
+  const client = new sdk.Client();
+
+  const storage = new sdk.Storage(client);
+
+  // initialize appwrite client
+  if (
+    !req.variables['APPWRITE_FUNCTION_ENDPOINT'] ||
+    !req.variables['APPWRITE_FUNCTION_API_KEY']
+  ) {
+    console.warn("Environment variables are not set. Function cannot use Appwrite SDK.");
+  } else {
+    client
+      .setEndpoint(req.variables['APPWRITE_FUNCTION_ENDPOINT'])
+      .setProject(req.variables['APPWRITE_FUNCTION_PROJECT_ID'])
+      .setKey(req.variables['APPWRITE_FUNCTION_API_KEY']);
+  }
+
+  // parse payload
+  const payload = req.payload;
+  const {bucketId} = JSON.parse(payload);
+  if (!bucketId) {
+    console.warn("Bucket ID is not set.");
+    return;
+  }
+
+  // get files
+  let files;
+  try {
+    files = await storage.listFiles(bucketId);
```

Comment:
By default, [`storage.listFiles()`](https://appwrite.io/docs/server/storage?sdk=nodejs-default#storageListFiles) returns the first 25 results. As such, this will only delete the 1st 25 files in the bucket. See our [Pagination](https://appwrite.io/docs/pagination) docs for more info.

Reply:
That makes sense, but I think a little note about that [in the docs](https://appwrite.io/docs/client/storage?sdk=web-default#storageListFiles) would be nice to have.

Reply:
That makes sense, but I think a little note about that [in the docs](https://appwrite.io/docs/client/storage?sdk=web-default#storageListFiles) would be nice to have.